webrender/src/device.rs

```diff
@@ -341,6 +341,8 @@ impl TextureId {
             target: gl::TEXTURE_2D,
         }
     }
+
+    pub fn is_invalid(&self) -> bool { *self == TextureId::invalid() }
```

Comment:
I originally had is_valid, but all call sites were negating it ```if !texture.is_valid() { ...``` so I ended up bundling the negation in the function. I don't mind either way, whichever you prefer works for me.